packages/datadog-plugin-mocha/src/index.js

```diff
@@ -50,6 +51,17 @@ function createWrapRunTest (tracer, testEnvironmentMetadata, sourceRoot) {
 
       const { childOf, resource, ...testSpanMetadata } = getTestSpanMetadata(tracer, this.test, sourceRoot)
 
+      const testParamsList = nameToParams[this.test.title]
+      if (Array.isArray(testParamsList)) {
+        try {
```

Comment:
Looks like. this code is duplicated between Jest and Mocha, any way to make it reusable?

Reply:
yeah, I'll update this after merging https://github.com/DataDog/dd-trace-js/pull/1345